### PR TITLE
Adjust events polling filter to handle edge case

### PR DESCRIPTION
**Pull request by johnwcallahan**
### The problem:
In the new LKE Node Pool display, I noticed that after creating a new cluster the Node Rows weren’t always updated with the Linode’s status once provisioned. A few of the nodes might be displayed as "Powered off", even though the Linodes were running.

### The cause:
After studying the polled requests to `/events` during this sequence, I realized this was happening because _we weren't receiving all events from the API_. And if we don’t receive a completed `linode_boot` event, for example, we don’t update the Linode’s status. 

**So why aren’t we receiving all events from the API?** A little context on how we poll `/events`...

When we receive a batch of events, we find the most recent `created` time. We use that timestamp in a filter like this: 

```
{ "created": { "+gt": <mostRecentCreatedTime> } }
```
This means we won’t receive any events **created on or before the most recent created time**. If there are events still in progress, we add an "+or" clause, like this:

```
{
  "+or": [
    { "created": { "+gt": <mostRecentCreatedTime> } },
    { "+or": [{ "id":<inProgressEventId> }] }
  ]
}
``` 

This approach works fine for most cases. But as it turns out, **there is a potential dead zone.**

Consider the case where there are four events with the same `created` time (let’s say `2020-04-08T12:00:00`). It turns out, this is likely with something like LKE cluster provisioning.

If the request/query lands at _just the right time_, you may only see the first 3 of these events. Since subsequent `/events` requests have a filter on `created` being greater than `2020-04-08T12:00:00`, we’ll never see the 4th event! (unless the app is refreshed). **This means the UI might not update appropriately.**

### My solution:
To solve this, I’ve changed the "+gt" filter to "+gte". Any (non-in-progress) events created at the mostRecentCreated time will be excluded via "+neq" filter.

### Bottom line:
Instead of polling `/events` with a filter like this:

```
{ "created": { "+gt": <mostRecentCreatedTime> } }
```

We poll with a filter that looks like this:

```
{
  "+and": [
    { "created": { "+gte": <mostRecentEventTime> } },
    { "id": { "+neq": <mostRecentEventId> } }
  ]
}
```

Check the new test cases for variations for in-progress events.

## Note to Reviewers

Most of the app should function like before. 

To see the bug that started this thing, provision a new LKE cluster with at least 6 nodes. You may have to do it a few times before you see the bug.

### CODE REVIEW

**Comment by MichaelSaah** [comment]
I think I understand the bug you're describing, but why is this necessary: `{ "id": { "+neq": <mostRecentEventId> } }`?

is it just to save the effort of retrieving and parsing that event which you know you've already seen?

**Comment by johnwcallahan** [comment]
> is it just to save the effort of retrieving and parsing that event which you know you've already seen?

Yeah, pretty much. Without it we'd be grabbing the same event(s) over and over from the API, and we'd have to change a lot of other logic that depends on events (or lack thereof) coming back after each request.
